notebooks/README.md

```diff
@@ -0,0 +1 @@
+The 'google_api' Notebook has the code to connect to Google Cloud Vision API and return the transcribed text extracted from a given image
```

Comment:
I really like the idea of documenting each file's function in the corresponding folder's README